### ISSUE TITLE
OPL-update doesn't respect site.conf for htdocs/DATA/*.json

### ISSUE THREAD

**Issue by ametts** [openwebwork/webwork2#1458]
In site.conf, I've pointed my htdocs and courses directories to a shared filesystem in a multi-server configuration.  But if I set my htdocs directory in site.conf as follows...

```
# URL and path to htdocs directory.
$webwork_htdocs_url  = "/webwork2_files";
$webwork_htdocs_dir  = <someplace-that-has-nothing-to-do-with-$WEBWORK_ROOT> ;
```
Running OPL-update will write tagging-taxonomy.json into $WEBWORK_ROOT/htdocs/DATA regardless of the site.conf settings.  This leads to "Could not load the OPL Taxonomy from the server" errors in the application.

Line ~~530~~ 546 of OPL-update seems to decide for itself where the webwork htdocs directory is:
```
#### Save the official taxonomy in json format
my $webwork_htdocs = $ce->{webwork_dir}."/htdocs";
my $file = "$webwork_htdocs/DATA/tagging-taxonomy.json";
```

In fact, all of the output JSON files for OPL-update behave in this manner, as does (apparently) the WebWork UI.   If the JSON files exist in my shared filesystem referenced as htdocs in site.conf, but not in the htdocs subdirectory of WEBWORK_ROOT (which is /opt/webwork/webwork2/htdocs on individual servers), they won't be found when using the UI.  In order to avoid running the lengthy OPL-update process on each server, I've had to set up symlinks of htdocs/DATA directory from each server to the shared file system..

**Comment by drgrice1** [comment]
Your statement that "all of the WeBWorK UI" behaves in the same manner is not true.  I just checked, and every instance where the UI uses the htdocs directory correctly utilizes the `$ce->{webworkDirs}{htdocs}` variable which is set from `$webwork_htdocs_dir` set in site.conf.  You are correct that the OPL-update script should also use that variable.

**Comment by pstaabp** [comment]
Fixed in #1655
